inlang/source-code/sdk/src/loadProject.ts

```diff
@@ -64,19 +64,15 @@ export async function loadProject(args: {
 	_capture?: (id: string, props: Record<string, unknown>) => void
```

Comment:
if you swap the overloads position the right intellisense pops up...

Reply:
removing this in the next step